packages/components/package.json

```diff
@@ -9,11 +9,12 @@
     "@types/jest": "^27.4.0",
     "@types/lodash": "^4.14.178",
     "@types/node": "^17.0.16",
-    "@types/react": "^17.0.43",
     "@types/react-dom": "^17.0.14",
+    "antd": "^4.19.3",
```

Comment:
It would be nice if we could just bring in antd-grid, or some other smaller lib. Antd is pretty big.
https://www.npmjs.com/package/antd-grid-layout

Reply:
That said, this could be a bit tricky; happy to make into a separate story for later.

Reply:
Antd is pretty big, but I'd trust webpack to shake off anything that isn't needed in the final distribution. We also use card.